sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql

```diff
@@ -151,6 +213,7 @@ mobile_data_google AS (
     submission_date = @submission_date
     AND country NOT IN ('RU', 'UA', 'BY', 'TR', 'KZ', 'CN')
     AND normalized_app_name IN ('Focus', 'Fenix', 'Fennec')
```

Comment:
I think we should leave this as is until we launch the app_name fix. Then, we can just fix all cases of Fennec in one go. Otherwise, we'll need to maintain the new app name for most tables and also specific OS and app_name filtering for this table